docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,30 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+## Notable Features
+
+### Puppet 4 Support
+
+Puppet 4 is now supported by Katello. For more information, visit [the Puppet upgrade page](http://www.katello.org/docs/nightly/upgrade/puppet.html)
+
+### Content Host Subscriptions Bulk Actions
+
+Users can now attach Subscriptions to multiple Content Hosts using the UI. 
+
```

Comment:
- Users can now attach/remove/auto-attach Subscriptions to multiple Content Hosts using the UI via bulk actions.
- Similar functionality is also available via hammer under 'hammer host subscriptions ` sub command